Code/Common/include/sitkImage.h

```diff
@@ -159,14 +199,17 @@ namespace simple
      */
     uint64_t GetNumberOfPixels( void ) const;
 
-    /** Get/Set the Origin
+    /** Get/Set the Origin in physical space
      * @{
      */
     std::vector< double > GetOrigin( void ) const;
     void SetOrigin( const std::vector< double > &origin );
     /** @} */
 
-    /** Get/Set the Spacing
+    /** Get/Set the Spacing of the Image as an std::vector .
+     *
+     * The spacing describes the physical size of each pixel. The
+     * length of the vector is equal to the dimension of the Image.
```

Comment:
The length of the vector can be equal or greater than the dimension of the image? Should the "greater" option be explicitly stated?